packages/consumption/test/modules/requests/itemProcessors/proposeAttribute/ProposeAttributeRequestItemProcessor.test.ts

```diff
@@ -535,6 +624,75 @@ describe("ProposeAttributeRequestItemProcessor", function () {
             });
         });
 
+        test("throws an error when another RelationshipAttribute with same key was queried", async function () {
```

Comment:
You could add a test trying to accept a RelationshipAttribute with the same key without mustBeAccepted. Same for ReadAttributeRequestItemProcessor.